src/Linux/Host/Broker/Impl/Services/Utility.cs

```diff
@@ -46,15 +46,15 @@ public class Utility {
             return proc;
         }
 
-        public static bool AuthenticateUser(ILogger<IAuthenticationService> logger, IProcessServices ps,  string username, string password, out string profileDir) {
+        public static bool AuthenticateUser(ILogger<IAuthenticationService> logger, IProcessServices ps,  string username, string password, string allowedGroup, out string profileDir) {
```

Comment:
That is actually called by `IAuthenticationService` implementation for Linux, i.e., `LinuxAuthenticationService`.